waiter/src/waiter/util/client_tools.clj

```diff
@@ -315,6 +315,7 @@
    :health-check-url "/status"
    :idle-timeout-mins 10
    :mem 256
+   :metric-group "waiter_kitchen"
```

Comment:
Should we instead specify `waiter_test` so we can isolated metrics from running our tests later?

Reply:
Renamed to `waiter_kitchen_test`.